pkgs/development/roswell/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, curl, autoreconfHook, makeWrapper, sbcl}:
+
+stdenv.mkDerivation rec {
+  pname = "roswell";
+  version = "21.01.14.108";
+  src = fetchFromGitHub {
+    owner = "roswell";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hj9q3ig7naky3pb3jkl9yjc9xkg0k7js3glxicv0aqffx9hkp3p";
+  };
+
+  configurePhase = ''
+    runHook preConfigure
+    sh bootstrap && ./configure --prefix $out
+    runHook postConfigure
+  '';
```

```suggestion
  preConfigure = ''
    sh bootstrap
  '';

  configureFlags = [ "--prefix=${placeholder "out"}" ];
```

@SuperSandro2000 `$out` -> `${placeholder "out"}` in flags, otherwise you get garbage results.

> `/nix/store/dyikz835agazxvxm9fj7xqlsz0k34iwj-coreutils-8.32/bin/mkdir -p 'ut/bin'`

---

that must have been it :]
thanks